### DISCUSSION TITLE
sumneko_lua is deprecated, use lua_ls instead.

### DISCUSSION THREAD

**Discussion by busyhe** [nshen/learn-neovim-lua#169] [Q&A]
sumneko_lua is deprecated, use lua_ls instead.
This feature will be removed in lspconfig version 0.2.0
...art/nvim-lsp-installer/lua/nvim-lsp-installer/server.lua:90: attempt to index field 'manager' (a nil value)

**Comment by TecFancy** [comment]
自己换了就行了

**Comment by nshen** [comment]
lua 的 lsp 改名了改成 lua_ls 了
最新的配置参考 https://github.com/nshen/InsisVim
